EarlGrey/Assertion/GREYAssertionDefines.h

```diff
@@ -107,6 +107,48 @@ GREY_EXTERN id<GREYFailureHandler> greyFailureHandler;
   I_GREYAssertEqual(__a1, __a2, __reason, ##__VA_ARGS__)
 
 /**
+ *  Generates a failure with the provided @c __reason if the @c __a1 and @c __a2 are equal.
```

Comment:
if the "expression" @c __a1

And if you could please make that change in the comments above, it'd be great. thanks.